### PR TITLE
clean up syntax

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: _includes/cwl/15-staging/linkfile.cwl [hunk 1]
```diff
@@ -8,8 +8,7 @@ hints:
 baseCommand: javac
 
 requirements:
-  - class: InlineJavascriptRequirement
-  - class: InitialWorkDirRequirement
+  InitialWorkDirRequirement:
```
Comment (tom-tan) [inline _includes/cwl/15-staging/linkfile.cwl]: This syntax is not valid in v1.0.2 because the spec says that `requirements` should be an array.

Comment (mr-c) [reply _includes/cwl/15-staging/linkfile.cwl]: In the spec itself a mapping using the value of the `class` field as the key is allowed: https://github.com/common-workflow-language/common-workflow-language/blob/master/v1.0/Process.yml#L811 but I see that it isn't rendering in the HTML version: http://www.commonwl.org/v1.0/CommandLineTool.html#CommandLineTool I also reviewed all 10 captures of this page in archive.org across 2016 & 2017 and the same rendering bug is present.

@tetron Any ideas?

Comment (mr-c) [reply _includes/cwl/15-staging/linkfile.cwl]: I've opend an issue about this at https://github.com/common-workflow-language/schema_salad/issues/167 and I think it is rather important to fix.

Comment (tom-tan) [reply _includes/cwl/15-staging/linkfile.cwl]: @mr-c I got it. Thanks for the information!

**Pull request merged**